src/components/SchedulePanel/components/RemoveHoursDialog/index.js

```diff
@@ -0,0 +1,157 @@
+import React, { Fragment } from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogActions from '@material-ui/core/DialogActions';
+import Button from '@material-ui/core/Button';
+//import Box from '@material-ui/core/Box';
+import { gordonColors } from '../../../../theme';
+
+export default class RemoveHoursDialog extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      dayOfTheWeek: '',
+      startHour: '',
+      endHour: '',
+      hourType: '',
+      // facebookInput: '',
+      // twitterInput: '',
+      // linkedInInput: '',
+      // instagramInput: '',
+      // formErrors: {
+      //   facebookInput: '',
+      //   twitterInput: '',
+      //   linkedInInput: '',
+      //   instagramInput: '',
+      // },
+      // fbValid: true,
+      // twValid: true,
+      // liValid: true,
+      // igValid: true,
+      // formValid: true,
```

Comment:
If none of the social media logic from lines 18-32 are used, we should remove. Unless they're placeholders for features that have yet to be implemented.